src/config.ts

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AuthenticationConfig, ServiceConfig} from
+    '../third_party/types/common-types';
+
+// ProfileAgentConfig is the configuration for the profiler.
+export interface ProfileAgentConfig extends AuthenticationConfig {
+  // Log levels: 0-disabled,1-error,2-warn,3-info,4-debug.
+  // Defaults value of 1.
```

Comment:
@GoogleCloudPlatform/node-team, readability reviewers, any thoughts on a better way to do defaults with TypeScript? Right now we have a copy of `defaultConfig` with the actual values that are also copied here.

Reply:
My thought is that we shouldn't mention default values in the interface definition at all, and include the default config at the bottom of this file ([like in the trace agent, currently](https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/blob/master/config.ts)). AFAIK, there isn't a clean way to include the type definition *and* default value in the same place.

Reply:
As a reader I'd prefer to have the documentation on the fields and their default values in one place. Having to jump up and down seems a bit strange.